olm-catalog/serverless-operator/1.7.0/serverless-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -376,6 +378,8 @@ spec:
                       value: registry.svc.ci.openshift.org/openshift/knative-v0.13.0:knative-eventing-filter
                     - name: IMAGE_DISPATCHER_IMAGE
                       value: registry.svc.ci.openshift.org/openshift/knative-v0.13.0:knative-eventing-channel-dispatcher
+                    - name: IMAGE_KN_CLI_ARTIFACTS
+                      value: registry.svc.ci.openshift.org/openshift/knative-v0.13.2:kn-cli-artifacts
```

Comment:
@warrenvw : please note the co-ordinates we'd need to replace with productized kn-cli-artifacts image.